doc/api/vm.md

```diff
@@ -409,76 +409,74 @@ loader][]. There is also no way to interact with the Loader yet, though
 support is planned.
 
 ```js
-const vm = require('vm');
```

Comment:
It should be emphasized somewhere in here that this is ESM syntax.

(side note: I'd love it if our docs could be improved to show both flavors... much like some other projects do -- see the below screenshot from fastify.io -- but that's definitely a much larger type of change) 

![image](https://user-images.githubusercontent.com/439929/106012690-7e823c00-6070-11eb-907e-c3e29bfaff3a.png)

Reply:
Both flavors would be ideal, I agree. What do you have in mind for the time being? Adding a comment? A foot-note?

Reply:
Also, does anyone has an opinion if we should promote the `node:` URL scheme (`import 'node:vm'`) instead?

Reply:
> Also, does anyone has an opinion if we should promote the `node:` URL scheme (`import 'node:vm'`) instead?

I think we probably should do that.

---

But as @GeoffreyBooth said, it should probably be done in its own PR that updates all the docs at once: <https://github.com/nodejs/node/pull/37170#discussion_r568090818>.

Reply:
Addressed :)